package.json

```diff
@@ -128,7 +128,7 @@
     "vite-plugin-pwa": "^0.14.0",
     "vite-plugin-static-copy": "^0.13.0",
     "vite-plugin-stylelint": "^4.0.1",
-    "vitest": "^0.32.3",
+    "vitest": "^0.33.0",
```

Comment:
I'm resolving this conversation since the comment has become obsolete because `vitest` is already updated on `develop` and the latest changes have been merged in this branch and PR.